elytron/WFCORE-4750-regex-role-mapper.adoc

```diff
@@ -0,0 +1,70 @@
+= WFCORE-4750 Using regex for role in Elytron
+:author:            Diana Vilkolakova
+:email:             dvilkola@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This task is is about introducing the possibility of role mapping based on regex in Elytron. With this functionality it will be possible for customers to easily translate list of roles (eg. *-admin, *-user) to simpler roles (eg. admin, user) without having to implement their own custom components to achieve this.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFCORE-4750[WFCORE-4750]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1344[EAP7-1344]
+
+=== Dev Contacts
+
+* mailto:dvilkola@redhat.com[Diana Vilkolakova]
+
+=== QE Contacts
+
+* TODO
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE.
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
```

Comment:
Plan to add tests with everything you do but we may still want to end up with QE testing before merge as this has a reasonable priority.

Reply:
Fixed